### PR TITLE
help options now display man pages

### LINKED ISSUE #14: individual command help doesn't work
```
aboyd@aboyd-laptop ~ $ vaulted shell --help
Usage of shell:
pflag: help requested
```

and similar for all other commands. This is somewhat less than helpful.

### PR DESCRIPTION
**Pull request by ryan-norton**
fixes #14

### CODE REVIEW

File: args_test.go [hunk 1]
```diff
@@ -135,11 +155,31 @@ var (
 				Shell:     "fish",
 			},
 		},
+		{
+			Args:    []string{"env", "--help"},
+			Command: &Help{Subcommand: "env"},
+		},
 
 		// Help
```
Comment (miquella) [inline args_test.go]: You appear to be missing test cases for some of the subcommands: `env`, `ls`, `list`, `load`, `rm`, `shell`, and `upgrade`.

File: args_test.go [hunk 2]
```diff
@@ -135,11 +155,31 @@ var (
...
+		},
+		{
+			Args:    []string{"help", "copy"},
+			Command: &Help{Subcommand: "copy"},
+		},
+		{
+			Args:    []string{"help", "edit"},
+			Command: &Help{Subcommand: "edit"},
+		},
+		{
```
Comment (miquella) [inline args_test.go]: Please sort the test cases lexicographically (dump before edit).

File: args_test.go (hunk 1 above)
Comment (ryan-norton) [inline args_test.go]: Weird, I recalled adding these, they may have fallen off at some point.  Added.

File: args_test.go (hunk 2 above)
Comment (ryan-norton) [inline args_test.go]: Fixed

**Pull request merged**